lib/utils/index.js

```diff
@@ -60,6 +60,7 @@ export {
 	watchForDescendants,
 	watchForFutureDescendants,
 	waitForEvent,
+	findElement,
```

Comment:
could you alphabetize this (and `empty` too)?

Reply:
Sure.